### DISCUSSION TITLE
There is no way to draw lines that cross 180th meridian in deckgl?

### DISCUSSION THREAD

**Discussion by xli1024** [visgl/deck.gl#5905] [Q&A] [answered]
Looking to do something like this with arc layer: https://docs.mapbox.com/mapbox-gl-js/example/line-across-180th-meridian/

It turns out that any line that cross 180th meridian will always get split into two segments and we can't have a map view that shows a complete line between US and Asia when plot on a map.

Is there anyway to implement the drawing like mapbox example above? Thanks in advance!

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/api-reference/core/map-view#repeat

**Reply by xli1024** [reply]
Thank you so much for the reply, worked like a charm! 感谢！
